app/controllers/application_controller.rb

```diff
@@ -1,13 +1,33 @@
 class ApplicationController < ActionController::Base
-  helper_method :current_user
-
-  def require_user_logged_in!
-    redirect_to root_path, notice: "You must be signed in to do that." if current_user.nil?
-  end
-
-  private
- 
-  def current_user
-    @current_user ||= User.find(session[:user_id]) if session[:user_id]
-  end
-end
+    skip_before_action :verify_authenticity_token
```

Comment:
This disables authenticity token verification on all actions in the app, which opens you up to CSRF attacks.  You'll want to remove this.